guerravc/KChallenge
# Feature One Listing Most Popular Movies

#### Purpose

#### Details
This feature let to the user see most popular movies

#### Steps to Test
* Run the app.
*  Scroll into de table, you can see movies.

#### Screenshots
![ListMoviesView](https://github.com/user-attachments/assets/1315dde7-9051-4da2-a297-317429a80a7b)
